src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -182,3 +187,35 @@ export const getAssertPermissionFn = (level: PermissionLevel): AssertFormFn => {
       return assertUnreachable(level)
   }
 }
+
+/**
+ * Reshapes given duplicate params into override props.
+ * @param params the parameters to reshape
+ * @param newAdminId the new admin id to inject
+ *
+ * @returns override props for use in duplicating a form
+ */
+export const processDuplicateOverrideProps = (
+  params: DuplicateFormBody,
+  newAdminId: string,
+): OverrideProps => {
+  const { responseMode, title } = params
+
+  const overrideProps: OverrideProps = {
+    responseMode,
+    title,
+    isNew: true,
```

Comment:
is this necessary?

Reply:
awesome, checked and seems to be legacy code! Thanks for the catch!